apps/server/src/modules/tool/uc/school-external-tool.uc.spec.ts

```diff
@@ -239,4 +239,65 @@ describe('SchoolExternalToolUc', () => {
 			});
 		});
 	});
+
+	describe('updateSchoolExternalTool is called', () => {
+		const setupUpdate = () => {
+			const { tool, user } = setup();
+			const updatedTool: SchoolExternalToolDO = { ...tool };
+			updatedTool.parameters[0].value = 'updatedValue';
+
+			schoolExternalToolService.saveSchoolExternalTool.mockResolvedValue(updatedTool);
+			return {
+				updatedTool,
+				schoolExternalToolId: updatedTool.id as EntityId,
+				user,
+			};
+		};
+
+		describe('when userId, schoolExternalToolId and schoolExternalTool are given', () => {
```

Comment:
This describe-block is still unnecessary. As a rule of thumb: You only need an extra describe-block if it has its own setup.